app/models/financialDetails/DocumentDetail.scala

```diff
@@ -160,6 +160,11 @@ case class DocumentDetail(taxYear: Int,
     case _ => false
   }
 
+  val isPOA: Boolean = {
+    val poaDescriptions = Seq("ITSA- POA 1", "ITSA - POA 2")
```

Comment:
:question: - not sure how, but I thinking if we can move 
```
  private val POA1: String = "ITSA- POA 1"
  private val POA2: String = "ITSA - POA 2"
```
in some sort of constants ie. we can have ClaimToAdjustConstants trait ? not sure,